### DISCUSSION TITLE
hypercore with curl

### DISCUSSION THREAD

**Discussion by ghost** [curl/curl#9595] [Q&A] [answered]
Hi all.

I would like to know if it would be possible to have the [hypercore](https://hypercore-protocol.org/) protocol in curl. *So... would it be possible to have the hypercore protocol in curl?*

**Note:** I have this doubt, because I'm using curl for backend and I use most of the supported protocols.

**What is Hypercore?**
"Hypercore Protocol is a peer-to-peer data network built on the Hypercore logs. Hypercores are signed, append-only logs. They're like lightweight blockchains without the consensus algorithm. As with BitTorrent, as more people "seed" a dataset it will increase the available bandwidth."

**Comment by bagder** [comment, marked as answer]
[Adding a new protocol?](https://curl.se/dev/new-protocol.html) contains some initial questions and guidelines.
